contracts/tester/src/types.rs

```diff
@@ -35,4 +35,30 @@ pub enum QueryMsg {
     /// `db_remove`, or `db_remove_range` FFI import methods directly. We need
     /// to test whether the VM can properly reject this behavior.
     ForceWrite { key: String, value: String },
+
+    CryptoVerify {
+        ty: CryptoVerifyType,
+        pk: Vec<u8>,
+        sig: Vec<u8>,
+        msg_hash: Vec<u8>,
```

Comment:
Don't use `Vec<u8>` in messages because they get serialized into Json arrays, e.g.

```json
{
  "pk": [
    1,
    2,
    3,
    4,
    5
  ]
}
```

Use `Binary` (if length is unknown) or `ByteArray<N>` (if length is known) instead. Here these should be `ByteArray<N>`.